content/en/docs/ops/troubleshooting/istioctl-analyze/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Analyze your cluster and local configuration with istioctl analyze
+description: Shows you how to use istioctl analyze to identify potential issues with your configuration.
+weight: 90
+keywords: [istioctl, debugging, kubernetes]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Getting started in under a minute
+
+`Istioctl analyze` is a powerful Istio diagnostic command that you can get started with in no time, and on any cluster.
+
+First, download the latest `istioctl` into the current folder using one bash command:
+
+Mac:
+
+{{< text bash >}}
+$ curl https://storage.googleapis.com/istio-build/dev/latest | xargs -I {} curl https://storage.googleapis.com/istio-build/dev/{}/istioctl-{}-osx.tar.gz | tar xvz
+{{< /text >}}
+
+Linux:
+
+{{< text bash >}}
+$ curl https://storage.googleapis.com/istio-build/dev/latest | xargs -I {} curl https://storage.googleapis.com/istio-build/dev/{}/istioctl-{}-linux.tar.gz | tar xvz
+{{< /text >}}
+
+Then, run it against your current Kubernetes cluster:
+
+{{< text bash >}}
+$ ./istioctl x analyze -k
+{{< /text >}}
+
+And that’s it! It’ll give you any recommendations that apply.
+
+For example, if you forgot to enable Istio injection (very common issue), you would get the following warning:
+
+{{< text plain >}}
+Warn [IST0102](Namespace/default) The namespace is not enabled for Istio injection. Run 'kubectl label namespace default istio-injection=enabled' to enable it, or 'kubectl label namespace default istio-injection=disabled' to explicitly mark it as not needing injection
+{{< /text >}}
+
+Notes: the ‘x’ in the command is because it’s currently ‘eXperimental’. It will eventually just be `istioctl analyze`.
+
+## Analyzing live clusters, local files, or both
+
+The scenario in the ‘getting started’ section is doing analysis on live cluster. But the tool also supports performing analysis of a set of local yaml configuration files, or on a combination of local files and a live cluster.
+
+{{< text bash >}}
+$
+# Analyze a specific set of local files
+$ ./istioctl x analyze a.yaml b.yaml
+
+# Analyze all yaml files in the current folder
+$ ./istioctl x analyze *.yaml
+
+# Analyze the current live cluster, simulating the effect of applying additional yaml files
+$ ./istioctl x analyze -k a.yaml b.yaml
+{{< /text >}}
+
+You can run `./istioctl x analyze --help` to see the full set of options.
+
+## Helping us make improve this tool
+
+This tool works by running a set of analyzers that can each detect a certain set of problems. While we’re working hard to expand the list of analyzers, we don’t necessarily have awareness of all the areas of Istio that can benefit from it.
```

Comment:
```suggestion
We're constantly adding more analysis capability and we'd love your help in identifying more use cases. If you've discovered some Istio configuration "gotcha", some tricky situation that caused you some problems, open an issue and let us know. We might be able to automatically flag this problem so that others can discover and avoid the problem in the first place.
```